blueprint/blueprint.h

```diff
@@ -12,7 +12,7 @@
     description:        A React.js render backend targeting JUCE Components.
     minimumCppStandard: 17
     dependencies:       juce_gui_basics
-    searchpaths:        ./duktape/src-noline/ ./yoga
+    searchpaths:        ./ ./duktape/ ./duktape/src-noline/ ./yoga
```

Comment:
The pathing here really didn't make sense... Duktape would never have been found; only its child files were found!